tools/tests/test_representations.py

```diff
@@ -562,6 +610,8 @@ def setUp(self):
             AUTHENTICATION_INFO: self.test_authentication_info_url,
             API_KEY_PARAMETER_NAME: self.test_api_key_parameter_name,
             LICENSE: self.test_license_url,
+            FEATURES: self.test_features,
+            STATUS: self.test_status,
         }
         self.test_schema = {
```

Comment:
? not sure what is missing.

Reply:
You added the `status` and the `features` to the `self.test_kwargs` above, but not to the `self.test_schema` below. This may cause errors when running the tests.

Reply:
thx for catching! fixed.